src/main/java/org/veupathdb/service/eda/ss/model/reducer/FormattedTabularRecordStreamer.java

```diff
@@ -85,7 +85,7 @@ public List<String> next() {
       if (valueStream.peek().getIdIndex() == currentIdIndex) {
         // TODO We need to a VariableType-Specific implementation of this toString.
```

Comment:
Still need this TODO?  Does getValue() always return String now?

Reply:
Thanks, good call. This is done now.